src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -402,7 +403,7 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 			const currentResponseData = element.response.value;
 			element.renderData ??= { renderedParts: [] };
 			const renderedParts = element.renderData.renderedParts;
-			const wordCountResults: IWordCountResult[] = [];
+			const wordCountResults: (IWordCountResult & { isPlaceholder: boolean })[] = [];
```

Comment:
I'm suspicious of this but it's ok

Reply:
You could look up `isPlaceholder` on the `currentResponseData` while rendering, right?

Reply:
Yeah, I guess I already index into two different arrays for this so a third isn't much worse